README.md

```diff
@@ -20,3 +20,19 @@
 Common developer tasks are available via `invoke` (defined in `tasks.py`)
 
 `invoke --list` to see available tasks.
+
+### Building and Running the GX Agent Image
+
+In order to to build the GX Agent docker image run the following in the root dir:
+
+```
+docker build -t gx/agent -f great_expectations_cloud/agent/Dockerfile .
```

Comment:
I would suggest putting this into an invoke task

Reply:
Added the invoke task and updated the readme, thanks for the suggestion!